src/config.ts

```diff
@@ -27,7 +27,7 @@ export default {
     HIGH_VALUE: {
       name: 'high value',
       description:
-        'Members with this role have access to more commands to moderate the server (e.g. ban). You get assigned this role if you have a message with +5 reactions',
+        'You get assigned the High Value role if you have a message with +5 reactions',
```

Comment:
You removed the sentence describing the permissions this role has. I am not sure if that is ideal.
```suggestion
        'You get assigned the High Value role if you have a message with +5 reactions. Members with this role have access to more commands to moderate the server (e.g. ban). ',
```
This, to me, feels better.

Reply:
Do you know what other permissions are included in that role?

Reply:
> You removed the sentence describing the permissions this role has. I am not sure if that is ideal.
> 
> This, to me, feels better.

No lol this is the problem, you dont get access this permission the description is wrong

Reply:
ohh .... lol

Reply:
Oh okay, then yeah it shouldn't be there.